client/go/outline/platerrors/platform_error.go

```diff
@@ -76,7 +88,10 @@ func (e PlatformError) Unwrap() error {
 // MarshalJSONString returns a JSON string containing the [PlatformError] details
 // and all its underlying causes.
 // The resulting JSON can be used to reconstruct the error in TypeScript.
-func MarshalJSONString(e PlatformError) (string, error) {
+func MarshalJSONString(e *PlatformError) (string, error) {
+	if e == nil {
```

Comment:
No need to change, but wouldn't it simplify it you change the return type to *string and return nil if the input is nil?

Reply:
Let me see what gomobile will do with `*string`.

Reply:
It is not recognized by gomobile either. gomobile only exports `string`, not `*string`.